flocker/node/agents/test/test_blockdevice.py

```diff
@@ -865,6 +1026,76 @@ class DiscoverVolumesMethod(Names):
     INFER_VOLUMES_FROM_STATE = NamedConstant()
 
 
+@implementer(ILocalState)
+class FakeBlockDeviceDeployerLocalState(PClass):
```

Comment:
Thanks for keeping the PR small :) I agree it doesn't make sense to bite off more testing work til calculate_changes is modified. And this may not be ideal, but you'll probably be deleting it then anyway.